components/accelerator/nvidia/query/infiniband/infiniband.go

```diff
@@ -12,24 +12,27 @@ import (
 	"github.com/leptonai/gpud/pkg/process"
 )
 
-// Returns true if the product supports infiniband.
-// e.g.,
-// "NVIDIA A100"
-// "NVIDIA H100"
-func SupportsInfinibandProduct(gpuProductName string) bool {
-	p := strings.ToLower(gpuProductName)
-	return strings.Contains(p, "a100") || strings.Contains(p, "h100")
-}
-
 // Returns the default non-zero per-port rate in GB/sec if the product supports infiniband.
+// ONLY used for "gpud scan". This is NOT and SHALL NOT be used for evaluating the infiniband component state.
```

Comment:
scan shall not check ib stats unless explicitly specified as well. revert for now.